### PR TITLE
[bitnami/several] Fix default values and regenerate README

### PR DESCRIPTION
**Pull request by pablogalegoc**
**Description of the change**

Fix default values in _values.yaml_ and regenerate README using the automated tool

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/logstash/values.yaml [hunk 1]
```diff
@@ -267,10 +267,10 @@ livenessProbe:
...
     port: monitoring
   initialDelaySeconds: 60
-  periodSeconds:
-  timeoutSeconds:
-  successThreshold:
-  failureThreshold:
+  periodSeconds: ""
+  timeoutSeconds: ""
+  successThreshold: ""
+  failureThreshold: ""
```
Comment (pablogalegoc) [inline bitnami/logstash/values.yaml:273]: This breaks the installation of logstash: https://github.com/bitnami/charts/pull/7045/checks?check_run_id=3145250534#step:6:1435

**Review by carrodher — APPROVED** [review]
LGTM

File: bitnami/logstash/values.yaml (hunk 1 above)
Comment (carrodher) [reply bitnami/logstash/values.yaml:273]: We usually set some default values for the probes, you can check other charts and apply the most used values

Comment (pablogalegoc) [reply bitnami/logstash/values.yaml]: Done in c5154d5 (I committed there a debug file by mistake, removed in 535e6e3). I've set the values to those that were suggested in comments before the changes to adapt the file to readme-generator:
https://github.com/bitnami/charts/blob/d6ba56d2fbc9e04728e7f39434213f5814ecf872/bitnami/logstash/values.yaml#L259-L262

**Pull request merged**